contents/product-engineers/mobile-app-metrics.md

```diff
@@ -0,0 +1,579 @@
+    imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/v1726495147/posthog.com/contents/Screenshot_2024-09-16_at_2.58.56_PM.png"
+    classes="rounded"
+    alt="Home screen bounce rate"
+/>
+
+### 3. Permissions granted rate
+
+**What it is:** The percentage of users who grant the necessary permissions to use your app.
+
+**Why it's useful:** Indicates if users are comfortable with your app and the permissions its requesting.
+
+**Good benchmark:** While there are no publicly available benchmarks, between 60-80% is considered good.
+
+**Questions to ask:**
+- Which permissions are most commonly granted, and which are least likely to be granted?
+- How does the number correlate with user activation and retention?
+- Which types of users ar more likely to trust your app and grant permissions?
+
+**Insight type:** [Funnel](/docs/product-analytics/funnels)
+
+<ProductScreenshot
+    imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/v1726495658/posthog.com/contents/Screenshot_2024-09-16_at_3.07.30_PM.png"
+    imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/v1726495488/posthog.com/contents/Screenshot_2024-09-16_at_3.04.39_PM.png"
+    classes="rounded"
+    alt="Permissions granted rate"
+/>
+
+### 4. Activation rate
+
+**What it is:** The percentage of users who reach the "aha" moment and experience your app's core value.
+
+**Why it's useful:** Measures how effective your app's onboarding and messaging is.
+
+**Good benchmark:** Above 8% – [Source](https://www.businessofapps.com/data/app-activation-rates/)
+
+**Questions to ask:**
+- Where are the largest drop-offs in the onboarding funnel before activation?
+- How does activation correlate with retention and customer lifetime value?
+- How do other external factors like marketing or app performance impact activation?
+
+**Insight type:** [Funnel](/docs/product-analytics/funnels)
+
+<ProductScreenshot
+    imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/v1726495892/posthog.com/contents/Screenshot_2024-09-16_at_3.11.24_PM.png"
+    imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/v1726495787/posthog.com/contents/Screenshot_2024-09-16_at_3.09.34_PM.png"
+    classes="rounded"
+    alt="Activation rate"
+/>
+
+### 5. Average time to activation
+
+**What it is:** The average time it takes for a user to reach activation.
+
+**Why it's useful:** Gives you a sense of how long it takes for a user to get value from your app.
+
+**Good benchmark:** Varies depending on the type of app.
+
+**Questions to ask:**
+- What are common characteristics of users who take shorter or longer to activate?
+- Is there a correlation between the time to activation and churn?
+- Is there a correlation between the speed of activation and long-term user retention?
+
+**Insight type:** [Funnel](/docs/product-analytics/funnels)
```

Comment:
There's a "time to conversion" visualization option for funnels
![Screenshot 2024-09-20 at 12 25 26](https://github.com/user-attachments/assets/39f49872-79f9-4aff-bf59-adcf2855874f)